styles.css

```diff
@@ -0,0 +1,788 @@
+body {
+    font-size: 16px;
+    line-height: 1.5;
+    color: #F5F8FB;
+    font-family: "Bakbak One", Arial, sans-serif;
+    background: #0D0D15;
+}
+
+/* Bakbak One for headings */
+.bakbak-one-regular {
+    font-family: "Bakbak One", sans-serif;
+    font-weight: 400;
+    font-style: normal;
+}
+
+/* Poppins for body text */
+.poppins-medium {
+    font-family: "Poppins", sans-serif;
+    font-weight: 500;
+    font-style: normal;
+}
+
+/* Header */
+.header {
+    padding: 32px;
+    /* Removed width: 100% as block elements automatically take full width */
+    background: #0D0D15;
+}
+
+.logo {
+    width: 188px;
+    height: 42px;
+}
+
+.logo img {
+    padding: 35px;
+    top: 1px;
+}
+
+.main-nav {
+    vertical-align: top;
+    width: auto;
+}
+
+.main-nav__list {
+    list-style: none;
+    padding: 0;
+    margin: 0;
+}
+
+.main-nav__link {
+    font-family: 'Poppins', sans-serif;
+    font-weight: 600;
+    font-size: 16px;
+    line-height: 1.5;
+    color: #F5F8FB;
+    text-decoration: none;
+}
+
+.main-nav__list-item {
+    display: inline;
+    margin-right: 20px;
+}
+
+.main-nav__link {
+    font-family: 'Poppins', sans-serif;
+    font-weight: 600;
+    font-size: 16px;
+    line-height: 1.5;
+    color: #F5F8FB;
+    text-decoration: none;
+}
+
+.main-nav__list-item {
+    font-family: 'Poppins';
+    font-weight: 600;
+    font-size: 16px;
+    line-height: 1.5;
+    color: #F5F8FB;
+}
+
+/* Hero Banner */
+.banner {
+    text-align: left;
+    padding: 50px 0;
+    background: radial-gradient(44.91% 42.19% at 22% -4%, rgba(2, 101, 220, 0.5), rgba(2, 101, 220, 0) 100%), rgb(13, 13, 21);
+    color: #F5F8FB;
+    font-size: 18px;
+    line-height: 1.5;
+}
+
+.banner__heading {
+    font-family: 'Bakbak One', sans-serif;
```

Comment:
If you set "Bakbak One" font for all page, u don't need to set it to all text elements.
Because for inheritance https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_cascade/Inheritance